UI/src/view/LinearAssetLayer.js

```diff
@@ -230,12 +243,24 @@ window.LinearAssetLayer = function(params) {
   };
 
   var changeTool = function(tool) {
-    if (tool === 'Cut') {
-      selectToolControl.deactivate();
-      linearAssetCutter.activate();
-    } else if (tool === 'Select') {
-      linearAssetCutter.deactivate();
-      selectToolControl.activate();
+    switch(tool) {
+      case 'Cut':
+        selectToolControl.deactivate();
+        linearAssetCutter.activate();
+        break;
+      case 'Select':
+        linearAssetCutter.deactivate();
+        selectToolControl.activate();
+        break;
+      case 'Rectangle':
+        linearAssetCutter.deactivate();
+        selectToolControl.activeDragBox();
```

Comment:
This name "selectToolControl.activeDragBox();" could be changed to "selectToolControl.activeRectangle()"